### PR TITLE
Salto-2846: Jira DC support Projects (part 3, last one)

### PR DESCRIPTION
**Pull request by IdoZakk**
_Plugin PR available [here](https://github.com/salto-io/jira-dc-app/pull/48)_

---

---
_Release Notes_: 
None

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/jira-adapter/src/filters/project.ts [hunk 1]
```diff
@@ -67,6 +67,10 @@ type ComponentsResponse = {
   }[]
 }
 
+const separateSchemeDeployment = (change: Change, isDataCenter: boolean): boolean =>
```
Comment (alonstern) [inline packages/jira-adapter/src/filters/project.ts:70]: super nit: `shouldSeparateSchemeDeployment`

File: packages/jira-adapter/src/filters/project_component.ts [hunk 2]
```diff
@@ -49,6 +49,10 @@ const filter: FilterCreator = ({ client, config }) => ({
         if (projectKey !== undefined) {
           instance.value.project = projectKey
         }
+        if (client.isDataCenter) {
+          instance.value.leadUserName = instance.value.leadAccountId
```
Comment (alonstern) [inline packages/jira-adapter/src/filters/project_component.ts:53]: Just for my knowledge, will `leadUserName`contains the id of the user or the username?

**Review by alonstern — APPROVED** [review]
🔥

File: packages/jira-adapter/src/filters/project_component.ts (hunk 2 above)
Comment (IdoZakk) [reply packages/jira-adapter/src/filters/project_component.ts:53]: The ID in DC is the user name AFAIK

**Pull request merged**